### DISCUSSION TITLE
目录标号问题

### DISCUSSION THREAD

**Discussion by ElegantNorlin** [reuixiy/reuixiy#16] [Q&A | 问答] [answered]
你好，我是meme主题的使用者，很喜欢这个主题，我现在遇到一个问题：
我使用Markdown进行写作，我一般喜欢使用三级标题作为第一季标题，但是上传到hugo上使用meme主题显示时，总是有一个默认的“1”目录号，而且我自己的目录号也都被打乱了，我描述的不是很清楚，麻烦您转到我的博客看一下吧
[https://elegantnorlin.github.io/](url)

希望得到作者大大的回复

**Comment by reuixiy** [comment, marked as answer]
尝试修改一下 https://github.com/reuixiy/hugo-theme-meme/blob/293d9a082903a16a686fd564c2310758e56cea95/config-examples/zh-cn/config.toml#L88

**Reply by ElegantNorlin** [reply]
感谢作者大大，问题解决啦！✌️
